content/en/docs/instrumentation/java/exporters.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Exporters
+weight: 50
+---
+
+In order to visualize and analyze your traces, you will need to export them to a
+backend such as [Jaeger](https://www.jaegertracing.io/) or
+[Zipkin](https://zipkin.io/). OpenTelemetry Java provides exporters for some
+common open source backends.
+
+If you use the Javaagent for
+[automatic instrumentation](/docs/instrumentation/java/automatic) you can learn
+how to setup exporters following the
+[Agent Configuration Guide](/docs/instrumentation/java/automatic/agent-config)
+
+For [manual instrumentation](/docs/instrumentation/java/manual), you will find
+some introductions below on how to set up backends and the matching exporters.
+
+## OTLP endpoint
+
+To send trace data to a OTLP endpoint (like the [collector](/docs/collector) or
+Jaeger) you'll want to use `opentelemetry-exporter-otlp`:
```

Comment:
Do you want this to be included in this PR or in a follow-up update?

Reply:
I'm happy to do in a followup. Just wanted to make the comment because its been a topic of conversation lately and I don't want it to get lost in the shuffle.

Reply:
Ok, let's keep this comment unresolved now and before merging this PR I'll create a follow-up issue

Reply:
follow up issue: #3271